src/views/IncorporationAgreement.vue

```diff
@@ -0,0 +1,316 @@
+          </span>
+        </v-tooltip>
+        containing a
+        <v-tooltip top color="primary">
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="tool-tip"> benefit provision</span>
+          </template>
+          <span>
+            Clauses in the Articles which specify the public benefits to be promoted by the benefit company and the
+            company’s commitment to promote those benefits and to conduct business in a responsible and sustainable
+            manner.
+          </span>
+        </v-tooltip>
+        for the company you are about to incorporate.
+      </p>
+
+      <!-- Help Section -->
+      <span class="help-btn" @click="helpToggle = !helpToggle">
+        <v-icon color="blue darken-2" style="padding-right: 5px">mdi-help-circle-outline</v-icon>
+        <span v-if="!helpToggle">Help with Incorporation Agreement and Articles</span>
+        <span v-else>Hide Help</span>
+      </span>
+      <section v-show="helpToggle" class="incorporation-agreement-help">
+        <h4>What is the Sample Incorporation Agreement and Benefit Company Articles?</h4>
+        <ul>
+          <li>
+            The Sample Incorporation Agreement and Benefit Company Articles is a template that you can use to create an
+            incorporation agreement and articles for your company. It uses all the standard provisions suggested by
+            legislation and also includes a place to specify the company’s benefit provision.
+          </li>
+          <li>
+            If you would like to customize any other provisions in the Articles, you cannot use this sample. We
+            recommend seeking professional assistance from a lawyer or accountant to help you prepare your Articles.
+          </li>
+        </ul>
+
+        <h4>What is a Benefit Provision?</h4>
+        <ul>
+          <li>
+            A Benefit Provision is a statement by the company of its public benefits and its commitments to promote
+            those public benefits and to conduct business in a responsible and sustainable manner.
+          </li>
+          <li>
+            A Benefit Company must include a benefit provision in its Articles.
+          </li>
+        </ul>
+        <h4>Can I use the Sample Incorporation Agreement and Benefit Company Articles?</h4>
+        <h4>
+          <span><v-icon color="green" class="article-stmt-icon">mdi-check</v-icon></span>
+          <span>You can use the sample Articles if:</span>
+        </h4>
+        <ul class="articles-statements">
+          <li>
+            <div>
+              There are no special rights or restrictions attached to any class or series of shares in the corporation’s
+              authorized share structure
+            </div>
+            <div>AND</div>
+          </li>
+          <li>
+            You do not wish to change any of the standard provisions in the sample Articles.
+          </li>
+        </ul>
+        <h4>
```

Comment:
As the previous heading element is an H2, this should be an H3.
https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks

We did this on the CreateShareStructure page as well and I'd like to see this fixed. I realize we're using the H4 elements to get different styling (font size). So what we should do is change our H4s to H3s and add CSS in base.scss to make the H3s look like we want. Thanks!

Reply:
Is it ok to style h3 as h4? or do you want a div with the required style?

Reply:
Sure, just style it globally. (h3 will look like h4)

Reply:
It's important to maintain heading rank, per link above. It's a matter of using the right element (h3).
You can style the h3 using any mechanism you want, but as this is really a project-wide style, doing it in base.scss is good.